demo/src/containers/Chat/Chat.tsx

```diff
@@ -27,6 +29,14 @@ export const Chat = () => {
     [sendMessage],
   );
 
+  const handleStartTyping = useCallback(() => {
```

Comment:
These are already callbacks in the hook, right? So they shouldn't need double-wrapping?